availability-store/src/store.rs

```diff
@@ -238,7 +226,7 @@ impl Store {
 	/// Add a set of chunks.
 	///
 	/// The same as `add_erasure_chunk` but adds a set of chunks in one atomic transaction.
-	/// Checks that all chunks have the same `relay_parent`, `block_data_hash` and `parachain_id` fields.
+	/// Checks that all chunks have the same `relay_parent`, `block_hash` and `parachain_id` fields.
```

Comment:
Maybe an accidental change? The `block_data` is a Polkadot term for the witness data accompanying a `CandidateReceipt`, which proves the blocks validity. This is not the same as the parachain block itself.

Reply:
Fixed.